src/LinkDotNet.NCronJob/RetryPolicies/ExponentialBackoffPolicyCreator.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Polly;
+
+namespace LinkDotNet.NCronJob;
+
+/// <summary>
+/// A policy creator that configures an exponential back-off retry policy.
+/// </summary>
+public partial class ExponentialBackoffPolicyCreator : IPolicyCreator, IInitializablePolicyCreator
```

Comment:
Does that have to be `public`?

Reply:
No, only if we want to give someone access to  it. Originally had an implementation in the Sample project that used it but I removed it to simplify things.